src/index.js

```diff
@@ -138,3 +139,49 @@ exports.major = function (version) {
     }
     return null;
 };
+/**
+ * @summary Check if a version is greater than another
+ * @name gt
+ * @public
+ * @function
+ *
+ * @description Returns true if versionA is greater than versionB.
+ * Valid semver versions are always weighted above non semver strings.
+ * Non-semver strings are compared alphabetically.
+ *
+ * @param {string|null} versionA - The version string to compare against
+ * @param {string|null} versionB - The version string to compare to versionA
+ *
+ * @returns {boolean} - true if versionA is greater than versionB, otherwise false.
+ *
+ * @example
+ * resinSemver.gt('2.0.5', '1.16.0'); //--> true
+ *
+ * resinSemver.gt('Resin OS 2.0.5', 'Resin OS 2.0.2+rev2'); //--> false
+ *
+ * resinSemver.gt('1.16.0', 'Resin OS 2.0.2+rev2'); //--> false
+ *
+ * resinSemver.gt('Resin OS 2.0.2', 'Resin OS 2.0.2'); //--> false
+ */
+exports.gt = function (versionA, versionB) {
+    if (versionA && !versionB) {
+        return true;
+    }
+    if (!versionA && versionB || !versionA && !versionB) {
+        return false;
+    }
+    versionA = normalize(versionA);
+    versionB = normalize(versionB);
+    var isAValid = semver.valid(versionA);
+    var isBValid = semver.valid(versionB);
+    if (!isAValid && !isBValid) {
+        return versionA > versionB;
+    }
+    if (!isAValid && isBValid) {
+        return false;
+    }
+    if (isAValid && !isBValid) {
+        return true;
+    }
+    return semver.gt(versionA, versionB);
```

Comment:
As in `lt` and `gte` - this should probably be defined in terms of the other operations